tools/explorer/tt_adapter/src/tt_adapter/mlir.py

```diff
@@ -404,9 +413,15 @@ def parse_ttnn_ttnn_layout(attr):
     memory_layout = layout.memory_layout_as_int
     if memory_layout is not None:
         result.append(
-            graph_builder.KeyValue(
-                key="memory_layout",
-                value=str(ttnn.TensorMemoryLayout(memory_layout)),
+            make_editable_kv(
+                graph_builder.KeyValue(
+                    key="Tensor Memory Layout",
+                    value=str(ttnn.TensorMemoryLayout(memory_layout)),
+                ),
+                editable={
+                    "input_type": "value_list",
+                    "options": [str(o) for o in ttnn.TensorMemoryLayout],
+                },
             )
         )
```

Comment:
I've tested specifically the `memory_layout` attribute and it is not returning the `editable` property to the UI 😕

Reply:
Hmm that is strange. For me the editable property exists and it is displayed as editable in the UI. See screenshot below:
<img width="1485" alt="Screenshot 2024-12-17 at 14 43 16" src="https://github.com/user-attachments/assets/44e35ba0-ccac-4c3e-87d5-0bde12632584" />

Reply:
The problem that i am hitting after this is that even when i change the value in the editable in the UI it is not sent in settings. Settings is equal to: ` {'optimizationPolicy': 'DF Sharding'}`

Reply:
So, the way it is working now, when you click the upload button, it will send the changes to the server and then clean up the changes.

Also, it is not sending the changes to either the `execute` command nor the `convert` command that comes after.

I'll make a change to send it on both commands

Reply:
@odjuricicTT please check this branch: https://github.com/tenstorrent/model-explorer/pull/19